storage/txcache/txListBySenderMap_test.go

```diff
@@ -171,5 +176,8 @@ func createTxListBySenderMap(numSenders int) txListBySenderMap {
 }
 
 func newSendersMapToTest() txListBySenderMap {
-	return newTxListBySenderMap(4, CacheConfig{})
+	return newTxListBySenderMap(4, CacheConfig{
```

Comment:
line 175 causes copy lock for returned myMap

Reply:
Fixed (returning pointers now).